utils/datautils.js

```diff
@@ -115,10 +115,19 @@ function getCounts() {
             }
         }
     );
-    const totalclaimed = db.Claim.aggregate(
+    var totalclaimed = db.Claim.aggregate(
         'bounty',
         'sum'
     );
+
+      if (!isNaN(accepted)) {
```

Comment:
The if case is is a double negative. You probably meant if (isNaN(x))

Reply:
The isNaN(x) case was always returning true for both the variables `accepted` and `totalclaimed`, I double checked. However, the `Number.isNaN(x)` returns the correct response. So `Number.isNaN(x)` is equivalent to `!isNaN(x)`. Hence this.

Reply:
If *accepted* is not a number, consider accepted is zero.

Reply:
https://stackoverflow.com/questions/25176459/is-number-isnan-more-broken-than-isnan


The reason isNaN() is "broken" is because, ostensibly, type conversions aren't supposed to happen when testing values. That is the issue Number.isNaN() is designed to address. In particular, Number.isNaN() will only attempt to compare a value to NaN if the value is a number-type value. Any other type will return false, even if they are literally "not a number", because the type of the value NaN is number. See the respective MDN docs for isNaN() and Number.isNaN().

If you simply want to determine whether or not a value is of the number type, even if that value is NaN, use typeof instead


Read this in the provided link. 
I think here is where the discrepancy lies. Again, using `Number.isNaN(x)` would be better I suppose

Reply:
Anyway your statement
>  So Number.isNaN(x) is equivalent to !isNaN(x)
is wrong on every level possible.